processing/src/test/java/org/apache/druid/query/aggregation/first/StringFirstAggregationTest.java

```diff
@@ -38,7 +39,7 @@
 
 import java.nio.ByteBuffer;
 
-public class StringFirstAggregationTest
+public class StringFirstAggregationTest extends NullHandlingTest
```

Comment:
Shouldn't this extend `InitializedNullHandlingTest` ? I think `NullHandlingTest` is an actual test... I always get confused between these 2

Reply:
You're right. I fixed it.